cmd/svcat/output/broker.go

```diff
@@ -23,6 +23,13 @@ import (
 	"github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 )
 
+func getBrokerScope(broker servicecatalog.Broker) string {
+	if broker.GetNamespace() != "" {
```

Comment:
is empty string already resolved to "default" at this point?

Reply:
Note that this isn't an actual v1beta1 broker, it's our custom-spun interface that spans the two types for svcat to use - this always returns "" for CSB's: https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/v1beta1/broker.go#L30

Reply:
👍